### DISCUSSION TITLE
V5 vs v4

### DISCUSSION THREAD

**Discussion by crashworldtk** [DistroAV/DistroAV#788] [General]
I could have sworn there was a v5 release which I downloaded about a week ago, now only the v4 releases are present. Is the v4 release only compatible with NDI4 and the v5 is supposed to support NDI5, pardon me for being confused. I also read somewhere about a “dummy” version. Can someone please clarify.

**Comment by Trouffman** [comment]
Hey @crashworldtk indeed this can be confusing.

NDI Runtime V5 are "backward-compatible" with NDI V4-focused codebase. That mean you can install the NDI V5 runtime, and still use the Obs-ndi ( that is using the ndi v4 codebase).

Only thing is : no features/improvments from the V5 will be available until this is developed for. > this is ongoing via the Rewrite Branch (aka not yet available !)

Cheers
